pkg/ddl/ddl_test.go

```diff
@@ -202,6 +202,18 @@ func TestIgnorableSpec(t *testing.T) {
 }
 
 func TestBuildJobDependence(t *testing.T) {
+	vers := []model.JobVersion{
+		model.JobVersion1,
+		model.JobVersion2,
+	}
+
+	for _, ver := range vers {
+		model.SetJobVerInUse(ver)
```

Comment:
by setting global `SetJobVerInUse`, you must revert back on finish, it's more simple to pass the version directly into `testBuildJobDependence`.

Reply:
those test is for DDL job queue which only works in version below 6.2.0, i think it's ok to leave them in job version1